foundations/html_css/the-box-model/the-box-model.md

```diff
@@ -21,24 +21,24 @@ Basically, every single thing on a webpage is a rectangular box. These boxes can
 
 ![boxes](https://cdn.statically.io/gh/TheOdinProject/curriculum/main/foundations/html_css/the-box-model/imgs/boxes.png)
 
-You can use the browser's inspector to add this css to this webpage if you want. Boxes in boxes!
+You can use the browser's inspector to add CSS to this webpage if you want. Boxes in boxes!
```

Comment:
`this` CSS makes sense here... it's talking about a specific code snippet.

Reply:
I wasn't sure if that was what was meant, because I didn't know what "this" referred to. I've changed it to "the CSS above" - does that make sense? I think it's more clear that way.

Reply:
yep  I like it